crates/oxc_linter/src/snapshots/uninvoked_array_callback.snap

```diff
@@ -0,0 +1,32 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: uninvoked_array_callback
+---
+
+  ⚠ Uninvoked array callback
+   ╭─[uninvoked_array_callback.tsx:1:1]
+ 1 │ const list = new Array(5).map(_ => {})
+   ·              ──────┬───── ─┬─
+   ·                    │       ╰── this callback function will not be invoked
```

Comment:
Nitpick, but should this point to the actual callback instead of just the `map` method?

Reply:
You'll get a huge diagnostic message if the callback spans multiple lines.

This once caught a bug in a JSX.

Reply:
Should I remove the "function"? -> "this callback will not be invoked"